src/components/MobileSideMenu/scss/Styles.scss

```diff
@@ -0,0 +1,46 @@
+$mobile-side-menu-content-width: 70%;
+
+$overlay-background-color: transparentize(#4e4949, 0.2);
+
+.mobile-side-menu {
+
+}
+
+.mobile-side-menu__overlay {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  bottom: -100%;
+  background-color: $overlay-background-color;
+  left: 0;
+  z-index: 99999;
+  animation: menu-overlay-slide 0.25s forwards;
+}
+
+@keyframes menu-overlay-slide {
+  100% { bottom: 0; }
+}
+
+.mobile-side-menu__content {
+  position: fixed;
+  z-index: 999999;
+  left: calc(-1 * #{$mobile-side-menu-content-width}) ;
```

Comment:
`right: 100%;` will have the same affect here, and is more clear what you're trying to accomplish (push the component off screen)

Reply:
done